### PR TITLE
[FSSDK-9744] chore: prepare  release 3.10.3

### PR DESCRIPTION
**Pull request by muzahidul-opti**
## Summary

- Upload task request issue fixed
- App privacy manifest file added
- Handle duplicate keys in experiment

## Test plan
- All previous test should pass

## Issues
- [FSSDK-9744](https://jira.sso.episerver.net/browse/FSSDK-9744)

### CODE REVIEW

File: .github/workflows/swift.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ on:
         description: release
 
 env:
-  VERSION: 4.0.0-beta
+  VERSION: 3.10.3
```
Comment (pulak-opti) [inline .github/workflows/swift.yml:20]: I think this is a patch update of a previous version. CI should run on the latest version. Or is there any specific reason to use the previous version here?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
 # Optimizely Swift SDK Changelog
 
+## 3.10.3
+November 30, 2023
+
+### Bug Fixes
+* Remove redundant post request body in upload task. ([#521](https://github.com/optimizely/swift-sdk/pull/521/)).
+
+### Ehancement
```
Comment (jaeopt) [inline CHANGELOG.md:9]:
```suggestion
### Enhancement
```

**Review by jaeopt — APPROVED** [review]
LGTM - a nit

File: .github/workflows/swift.yml (hunk 1 above)
Comment (jaeopt) [reply .github/workflows/swift.yml:20]: This is just for version numbering into the built SDK.

**Pull request merged**